### PR TITLE
[STLND-1235] Updated OIDC client in line with the latest standards

### PR DESCRIPTION
**Pull request by jaszczur171**

### CODE REVIEW

File: StellaNowSDK/src/Config/StellaNowConfig.cs [hunk 1]
```diff
@@ -27,4 +27,5 @@ public record StellaNowConfig
     public string OrganizationId { get; init; }
     public string ProjectId { get; init; }
     public string ClientId { get; init; }
+    public static string OidcClient => "tools-cli";
 }
```
Comment (wbar) [inline StellaNowSDK/src/Config/StellaNowConfig.cs:31]:
```suggestion
}

```

Comment (jaszczur171) [reply StellaNowSDK/src/Config/StellaNowConfig.cs:31]: This is not python :)

File: StellaNowSDKDemo/src/Program.cs [hunk 2]
```diff
@@ -74,6 +82,8 @@ private async Task RunAsync()
             
             // Delay between messages
             await Task.Delay(TimeSpan.FromSeconds(1));
```
Comment (wbar) [inline StellaNowSDKDemo/src/Program.cs:84]: Why hardcoding 1 second ?

Comment (jaszczur171) [reply StellaNowSDKDemo/src/Program.cs:84]: Its just a demo app, not part of the SDK itself

File: StellaNowSDK/src/Config/StellaNowConfig.cs (hunk 1 above)
Comment (wbar) [reply StellaNowSDK/src/Config/StellaNowConfig.cs:31]: NL at END of line - its GIT

**Review by wbar — APPROVED** [review]

**Comment by jaszczur171** [comment]
[STLND-1235] Updated OIDC client in line with the latest standards
- updated oidc client 
- version bump to 0.0.2

**Pull request merged**